tap_googleads/client.py

```diff
@@ -75,27 +70,9 @@ def http_headers(self) -> dict:
         if "user_agent" in self.config:
             headers["User-Agent"] = self.config.get("user_agent")
         headers["developer-token"] = self.config["developer_token"]
-        headers["login-customer-id"] = self.config.get("login_customer_id")
+        headers["login-customer-id"] = self.config["login_customer_id"]
```

Comment:
Ah yeah this is actually something I was hoping you guys would know. Its hard to test these different scenarios because I can only test the tap from a manager customer account. Let's go with this approach then, I will re-implement it.